chsdi/templates/htmlpopup/kgs.mako

```diff
@@ -17,7 +17,11 @@
         objarts = c['attributes']['objektart'].split(',')
         gemeinde_ehemalig = c['attributes']['gemeinde_ehemalig'] if c['attributes']['gemeinde_ehemalig'].startswith("(") else "("+c['attributes']['gemeinde_ehemalig']+")"
         import csv
-        from urllib2 import urlopen
+        try:
+            from urllib2 import urlopen
+        except ImportError:
+            # Python3 fallback
+            from urllib.request import urlopen
```

Comment:
The make does not work in py3 yet. 
/rest/services/ech/MapServer/ch.babs.kulturgueter/9980/extendedHtmlPopup?lang=en
``pyramid_mako.MakoRenderingException: 

Traceback (most recent call last):
  File "/home/ltrea/mf-chsdi3/.venv/lib/python3.7/site-packages/pyramid_mako/__init__.py", line 148, in __call__
    result = template.render_unicode(**system)
  File "/home/ltrea/mf-chsdi3/.venv/lib/python3.7/site-packages/mako/template.py", line 482, in render_unicode
    self, self.callable_, args, data, as_unicode=True
  File "/home/ltrea/mf-chsdi3/.venv/lib/python3.7/site-packages/mako/runtime.py", line 883, in _render
    **_kwargs_for_callable(callable_, data)
  File "/home/ltrea/mf-chsdi3/.venv/lib/python3.7/site-packages/mako/runtime.py", line 920, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/home/ltrea/mf-chsdi3/.venv/lib/python3.7/site-packages/mako/runtime.py", line 947, in _exec_template
    callable_(context, *args, **kwargs)
  File "/home/ltrea/mf-chsdi3/chsdi/templates/htmlpopup/base.mako", line 52, in render_body
    ${self.extended_info(c, lang)}
  File "/home/ltrea/mf-chsdi3/chsdi/templates/htmlpopup/kgs.mako", line 31, in render_extended_info
    next(reader) # Skip header
Error: iterator should return strings, not bytes (did you open the file in text mode?```

Reply:
@rebert but this has nothing to do with this PR. We can already merge this and fix the other python3 issues later on.